setup.py

```diff
@@ -17,10 +18,33 @@
 # You should have received a copy of the GNU General Public License
 # along with ShinySDR.  If not, see <http://www.gnu.org/licenses/>.
 
+# This setup.py is Based on the work from:
```

Comment:
In general: Put provenance information in commit description, not comments, except where it affects licensing.

In specific: I don't see that you copied anything substantial from there?

Reply:
agreed